### PR TITLE
533 refactor menu remove tim

### LINKED ISSUE #533: 🔧 [Refactor] Refactor menu - remove tim
## 🔧 Refactor ticket

The mobile menu has many elements.

Does the layout work on all devices?
Do we follow the standards?
Is the menu practical for our most common use case?

Task:
- Refactor Tim
- Simplify Menu Code
- Have 90%+ coverage

### LINKED ISSUE #955: 💥 [DevOps] Enforce 60% coverage (backend)
## 💥 DevOps ticket

Enforce 60% coverage (backend)

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
remove Navbar and Sidebar from creative tim template 
remove template creative tim und work 100% bootstrap

### Issues
- fixes #533
- fixes #955

### CODE REVIEW

File: frontend/src/components/Menu/Navbar.spec.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  describe('mount', () => {
+    beforeEach(() => {
+      wrapper = Wrapper()
+    })
+
+    it('renders the component', () => {
+      expect(wrapper.find('div.component-navbar').exists()).toBeTruthy()
+    })
+  })
```
Comment (Elweyn) [inline frontend/src/components/Menu/Navbar.spec.js:39]: I think that their is an error here, you don't want to close this here
```suggestion
```

File: frontend/src/components/Menu/Navbar.spec.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      expect(wrapper.findAll('.nav-item').at(7).find('a').attributes('href')).toBe('#')
+    })
+
+    it('has first nav-item "admin_area" in navbar', () => {
+      expect(wrapper.findAll('.nav-item').at(8).text()).toEqual('admin_area')
+    })
+    it('has first nav-item "logout" in navbar', () => {
+      expect(wrapper.findAll('.nav-item').at(9).text()).toEqual('logout')
+    })
+  })
```
Comment (Elweyn) [inline frontend/src/components/Menu/Navbar.spec.js:80]: The closing tag should be here (IMO)
```suggestion
  })
  })
```

File: frontend/src/components/Menu/Sidebar.spec.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  describe('mount', () => {
+    beforeEach(() => {
+      wrapper = Wrapper()
+    })
+    it('renders the component', () => {
+      expect(wrapper.find('div#component-sidebar').exists()).toBeTruthy()
+    })
+  })
```
Comment (Elweyn) [inline frontend/src/components/Menu/Sidebar.spec.js:33]: Same as in NavBar I think this closing tag should be at the end
```suggestion
```

File: frontend/src/components/Menu/Sidebar.spec.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      expect(wrapper.findAll('.nav-item').at(4).text()).toContain('members_area')
+      expect(wrapper.findAll('.nav-item').at(4).find('a').attributes('href')).toBe('#')
+    })
+    it('has first nav-item "admin_area" in navbar', () => {
+      expect(wrapper.findAll('.nav-item').at(5).text()).toEqual('admin_area')
+    })
+    it('has first nav-item "logout" in navbar', () => {
+      expect(wrapper.findAll('.nav-item').at(6).text()).toEqual('logout')
+    })
+  })
```
Comment (Elweyn) [inline frontend/src/components/Menu/Sidebar.spec.js:64]:
```suggestion
  })
  })
```

**Review by Elweyn — CHANGES_REQUESTED** [review]
Except the 4 changes that seems good, I like the view!

File: frontend/src/views/Layout/DashboardLayout_gdd.spec.js [hunk 5]
```diff
@@ -72,73 +72,23 @@ describe('DashboardLayoutGdd', () => {
...
-      })
-
-      it('has third item "transactions" linked to transactions in navbar', async () => {
-        expect(wrapper.findAll('ul > a > a').at(2).attributes('href')).toBe('/transactions')
-      })
-
-      it('has fourth item "My profile" in navbar', () => {
-        expect(navbar.findAll('ul > a').at(3).text()).toEqual('site.navbar.my-profil')
-      })
-
```
Comment (ulfgebhardt) [inline frontend/src/views/Layout/DashboardLayout_gdd.spec.js:125]: What happend to all of this? Why is the logout button still tested but the rest is misisng?

File: frontend/src/views/Pages/SendOverview.spec.js [hunk 6]
```diff
@@ -38,10 +38,6 @@ describe('SendOverview', () => {
       wrapper = Wrapper()
     })
 
-    it('has a status GDD line gdd-status-gdd', () => {
-      expect(wrapper.find('div.gdd-status-gdd').exists()).toBeTruthy()
-    })
-
```
Comment (ulfgebhardt) [inline frontend/src/views/Pages/SendOverview.spec.js:44]: We still have this right?

File: frontend/src/views/Pages/AccountOverview.vue [hunk 7]
```diff
@@ -16,7 +16,7 @@
         </b-col>
         <b-col class="col-6 text-right">
           <b-row>
-            <b-col class="bg-white text-gray p-3">
+            <b-col class="p-3" style="background-color: #f0f0f0">
```
Comment (ulfgebhardt) [inline frontend/src/views/Pages/AccountOverview.vue:19]: can we define this as style?

File: frontend/src/views/Pages/AccountOverview.vue [hunk 8]
```diff
@@ -1,10 +1,10 @@
 <template>
   <div>
-    <b-container fluid>
+    <div>
       <b-row>
         <b-col class="col-6">
           <b-row>
-            <b-col class="col-11 bg-gray text-white p-3">
+            <b-col class="col-11 ml-2 p-3" style="background-color: #f0f0f0">
```
Comment (ulfgebhardt) [inline frontend/src/views/Pages/AccountOverview.vue:7]: can we define this as style?

File: frontend/src/views/Layout/DashboardLayout_gdd.vue [hunk 9]
```diff
@@ -1,95 +1,65 @@
...
     return {
+      logo: 'img/brand/green.png',
       balance: 0,
       GdtBalance: 0,
       transactions: [],
       bookedBalance: 0,
       transactionCount: 0,
       pending: true,
+      visible: false,
+      elopageUri: this.getElopageLink(),
```
Comment (ulfgebhardt) [inline frontend/src/views/Layout/DashboardLayout_gdd.vue:62]: not sure this is good - @Mogge what you say?

**Review by ulfgebhardt — CHANGES_REQUESTED** [review]

File: frontend/src/views/Layout/DashboardLayout_gdd.spec.js (hunk 5 above)
Comment (ogerly) [reply frontend/src/views/Layout/DashboardLayout_gdd.spec.js:125]: dies wird in der Navbar.vue getestet

**Review by Elweyn — APPROVED** [review]

File: frontend/src/components/Menu/Navbar.spec.js [hunk 10]
```diff
@@ -81,4 +81,23 @@ describe('Navbar', () => {
       })
     })
   })
+  describe('check watch visible true', () => {
+    beforeEach(async () => {
+      await wrapper.setProps({ visible: true })
+    })
+
+    it('has visibleCollapse == visible', async () => {
```
Comment (Elweyn) [inline frontend/src/components/Menu/Navbar.spec.js:89]: No ``async`` needed here

File: frontend/src/components/Menu/Navbar.spec.js [hunk 11]
```diff
@@ -81,4 +81,23 @@ describe('Navbar', () => {
...
+      expect(wrapper.vm.visibleCollapse).toBe(true)
+    })
+  })
+
+  describe('check watch visible false', () => {
+    beforeEach(async () => {
+      await wrapper.setProps({ visible: false })
+    })
+
+    it('has visibleCollapse == visible', async () => {
```
Comment (Elweyn) [inline frontend/src/components/Menu/Navbar.spec.js:99]: No ``async`` needed here

**Review by Elweyn — APPROVED** [review]

**Pull request merged**